### PR TITLE
Backported #1616: Fix writing methods on read when reindexing services

### PR DESCRIPTION
**Pull request by ramonski**
## Description of the issue/feature this PR addresses

Backported #1616 to 1.3.x 

--
I confirm I have tested this PR thoroughly and coded it according to [PEP8][1]
and [Plone's Python styleguide][2] standards.

[1]: https://www.python.org/dev/peps/pep-0008
[2]: https://docs.plone.org/develop/styleguide/python.html

### CODE REVIEW

File: bika/lims/upgrade/v01_03_005.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+from bika.lims.config import PROJECTNAME as product
+from bika.lims.upgrade import upgradestep
+from bika.lims.upgrade.utils import UpgradeUtils
+
+version = "1.3.5"  # Remember version number in metadata.xml and setup.py
+profile = "profile-{0}:default".format(product)
+
+INDEXES_TO_ADD = [
+    # List of tuples (catalog_name, index_name, index meta type)
+    (CATALOG_ANALYSIS_REQUEST_LISTING, "modified", "DateIndex"),
```
Comment (xispa) [inline bika/lims/upgrade/v01_03_005.py]: I think this index addition does not belong to this PR

**Review by xispa — CHANGES_REQUESTED** [review]

File: bika/lims/upgrade/v01_03_005.py (hunk 1 above)
Comment (ramonski) [reply bika/lims/upgrade/v01_03_005.py]: Thanks, just removed that code snippet 👍

**Review by xispa — APPROVED** [review]
